job-server-extras/src/main/scala/spark/jobserver/python/PythonJob.scala

```diff
@@ -64,6 +64,7 @@ case class PythonJob[X <: PythonContextLike](eggPath: String,
         Process(
           Seq(sc.pythonExecutable, "-m", "sparkjobserver.subprocess", server.getListeningPort.toString),
           None,
+          "EGGPATH" -> eggPath,
```

Comment:
How are we going to upload the file to SJS node?

Reply:
the application EGG file would have already been uploaded using standard job binary API.  this patch ensures the already loaded app path is set in the context of the shelled spark instance.